app.js

```diff
@@ -0,0 +1,11 @@
+let calculators = ko.observableArray([new Calculator()]);
+
+function addCalculator() {
+  calculators.push(new Calculator());
+}
+
+function removeCalculator() {
+  calculators.remove(this);
+}
+
+ko.applyBindings(calculators);
```

Comment:
This is the most important piece of code that I have removed. The reason is that this code requires that a DOM exists (i.e. that the code is being tested in a browser). If this code runs, it will "bind" the calculators to the DOM. We want to test the calculator itself, not how it binds to the DOM.